DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
@jkarancs @davidlange6 it seems `DataFormats/SiPixelDetId` is not a feasible location since it would cause a recursive dependency via `Geometry/TrackerGeometryBuilder`.

In general, a package with the `SiPixelCoordinates`in it would  have quite massive dependencies. Which is not that big of a problem with the DQM package, since it has those dependencies either way...